app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -141,6 +141,33 @@ def bulk_cancel
         }
       end
 
+      api :POST, '/tasks/bulk_stop', N_('Stop all stoppable tasks')
+      param :search, String, :desc => N_('Stop tasks matching search string')
+      param :task_ids, Array, :desc => N_('Stop specific tasks by ID')
+      def bulk_stop
+        if params[:search].nil? && params[:task_ids].nil?
+          raise BadRequest, _('Please provide at least one of search or task_ids parameters in the request')
+        end
+
+        filtered_scope = bulk_scope
+        skipped, to_stop = filtered_scope.partition { |task| task.state == "stopped" }
+        to_stop.each { |task| task.update({ state: :stopped }) }
```

Comment:
You should be able to drop the curly braces around `state: :stopped`

Reply:
removed, thanks